src/EStimLibrary/Extensions/SpatialModel/StringHierarchy/StringHierarchyAreaFactory.cs

```diff
@@ -34,12 +34,12 @@ private bool AreaSpecCheckFunction(string fullSpec)
         var parts = fullSpec.Split(
             StringHierarchySpec.REGIONS_MODIFIERS_DELIMITER);
 
-        return parts.Length > 0 &&
+        return parts.Length < 3 &&
```

Comment:
For which cases was this change needed? What was failing? May be a larger bug/flaw